fusionSuite/frontend
# Tec/refacto

Changes proposed in this pull request:

- Introduce an Organization model to avoid using directly the `IItem`, it allows to use custom methods
- Split the API file in three: a file to implement the backend API, and the two others to abstract the use in the application

How to test the feature manually:

1. Use the application normally (create, list and delete organizations and users), everything should work normally

Pull request checklist:


_If you think one of the item isn’t applicable to the PR, please check it
anyway and precise `N/A` next to it._

Related to https://github.com/fusionSuite/FusionSuite/issues/42
